core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -418,8 +417,6 @@ class ReplicaManager(val config: KafkaConfig,
                    _: RecordTooLargeException |
                    _: RecordBatchTooLargeException |
                    _: CorruptRecordException |
-                   _: InvalidRecordException |
```

Comment:
Why were these two cases removed? Doesn't seem obvious to me why this would change.